doc/minutes/2022-12-21.md

```diff
@@ -0,0 +1,95 @@
+---
+title:  "Fourth meeting to organize the YOS award"
+date:   "21/12/2022"
+output: github_document
+editor_options:
+  markdown:
+    canonical: false
+---
+
+**Attendees:** Cristina Rodríguez, Daniel Morillo, Jesus Sierralaya, Lea
+Francia, Roberto Mediavilla
+
+**Scribe:** Daniel Morillo
+
+**Schedule:** 10:30 - 11:15
+
+# Action points
+
+## Completion of the form fields
+
+#### Debate
+
+- Changes in section "output" are pending
+
+- Branch `drafting/reusability` was incorrectly merged to `main` instead of
+  `drafting/template-fillin` as it was intended to
+
+#### Agreements
+
+-   **Approved**: Merge PRs to branch `drafting/template-fillin`
+
+#### Tasks
+
+-   **Roberto**: Review the remaining of the draft [ASAP]
+
+-   **Daniel**: Rebase branch `drafting/template-fillin` to `main` in order to
+    include the changes from `drafting/reusability` [ASAP]
+
+-   **Everyone**: Approve changes in the corresponding Github PRs [ASAP]
+
+-   **Roberto**: Submit form to Yerun Open Science Awards call [21/12/2022]
+
+## Team description section
+
+#### Debate
+
+-   Suggested that the "team description"" would better include a description of
```

Comment:
"team description", one " extra

Reply:
Thanks! Fixed in baf61bb8cfc2219a1ff0dfff6198db5a1844edd4